ctms/bin/acoustic_fields.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python3
+"""Manage Acoustic fields: add and remove from the db."""
+
+import argparse
+import sys
+
+from ctms import config
+from ctms.database import get_db_engine
+from ctms.models import AcousticField
+
+
+def main(dbsession, test_args=None) -> int:
+    parser = argparse.ArgumentParser(
+        description="""
+        Manage Acoustic fields
+        """
+    )
+    subparsers = parser.add_subparsers(dest="action")
+    parser_add = subparsers.add_parser("add")
+    parser_add.add_argument("field")
+    parser_add.add_argument(
+        "--tablename", "-t", help="Acoustic table name", default="main"
+    )
+
+    parser_remove = subparsers.add_parser("remove")
+    parser_remove.add_argument("field")
+    parser_remove.add_argument(
+        "--tablename",
+        "-t",
+        help="Acoustic table name (default: 'main')",
+        default="main",
+    )
+
+    parser_list = subparsers.add_parser("list")
+    parser_list.add_argument(
+        "--tablename",
+        "-t",
+        help="Acoustic table name (default: 'main')",
+        default="main",
+    )
+
+    args = parser.parse_args(args=test_args)
+    if args.action == "add":
+        dbsession.merge(AcousticField(tablename=args.tablename, field=args.field))
+        dbsession.commit()
+        print("Added.")
+    elif args.action == "remove":
+        row = (
+            dbsession.query(AcousticField)
+            .filter(
+                AcousticField.tablename == args.tablename,
+                AcousticField.field == args.field,
+            )
+            .one_or_none()
+        )
+        if not row:
+            print(f"Unknown field '{args.tablename}.{args.field}'. Give up.")
+            return 2
```

Comment:
Should we return one of the exit codes as defined in the `os` module? [E.g.](https://docs.python.org/3/library/os.html#os.EX_DATAERR)